<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolidRec_Dialog</class>
 <widget class="QDialog" name="SolidRec_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Solid Rectangle - Template</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	color: rgb(255, 255, 255);
	font: 9pt &quot;Segoe UI&quot;;
	background-color: rgb(43, 43, 43);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>t3.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,6">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="ModelingType_groupBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(128, 128, 128);</string>
         </property>
         <property name="title">
          <string>Modeling Type</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="Centerline_radioButton">
            <property name="text">
             <string>Centerline</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Outline_radioButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Outline</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="Dimension_groupBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(128, 128, 128);</string>
         </property>
         <property name="title">
          <string>Dimension</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="B_label">
              <property name="text">
               <string>B = </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="B_inputlineEdit">
              <property name="minimumSize">
               <size>
                <width>78</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="D_label">
              <property name="text">
               <string>D = </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="D_inputlineEdit">
              <property name="minimumSize">
               <size>
                <width>78</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="SolidRec_groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string>Material</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <item row="0" column="4">
         <widget class="QLabel" name="G_label">
          <property name="text">
           <string> E  = </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="ID_label">
          <property name="text">
           <string>Color = </string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="E_label">
          <property name="text">
           <string>ID = </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="fy_label">
          <property name="text">
           <string>fy  = </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Color_label">
          <property name="text">
           <string>     μ   = </string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="eu_inputlineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="fy_inputlineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="eu_label">
          <property name="text">
           <string>eu = </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="G_inputlineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="E_inputlineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="ID_inputlineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="ColorButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>19</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	border:1px solid rgb(0,0,0);
	background-color: rgb(0, 0, 0);
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(170, 255, 255);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="button_GroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(128, 128, 128);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="OK_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Cancel_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ID_inputlineEdit</tabstop>
  <tabstop>E_inputlineEdit</tabstop>
  <tabstop>G_inputlineEdit</tabstop>
  <tabstop>fy_inputlineEdit</tabstop>
  <tabstop>eu_inputlineEdit</tabstop>
  <tabstop>Centerline_radioButton</tabstop>
  <tabstop>Outline_radioButton</tabstop>
  <tabstop>B_inputlineEdit</tabstop>
  <tabstop>D_inputlineEdit</tabstop>
  <tabstop>OK_button</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
  <tabstop>ColorButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
